### ISSUE TITLE
SetClippingRegion fails

### ISSUE THREAD

**Issue by sancelot** [wxWidgets/Phoenix#1783]
**Operating system**: Win10
**wxPython version & source**:         4.1.0
**Python version & source**:            3.8.5

**Description of the problem**:

dc.setClippingRegion is failing with next message .

This was working with version 4.0.7.post2

e-type:
<class 'TypeError'>

e-value:
DC.SetClippingRegion(): arguments did not match any overloaded call:
  overload 1: too many arguments
  overload 2: argument 1 has unexpected type 'bool'
  overload 3: argument 1 has unexpected type 'bool'

locals:
    self: <wxSVGSelectionList.wxSVGSelectionListControl object at 0x000000001DC29DC0>
    clip: (0, 0, 337, 230)
    DrawFunc: <bound method wxSVGSelectionListControl.DrawControls of <wxSVGSelectionList.wxSVGSelectionListControl object at 0x000000001DC29DC0>>
    dc: <wx._core.BufferedPaintDC object at 0x000000001DC37280>
    ClippingRegion: (False, 0, 0, 1012, 740)

```python
           clip = (1,80,949,515)
            dc.SetClippingRegion(*clip)
```

**Comment by swt2c** [comment]
Do you have a full, runnable example that reproduces this problem?  At first glance, I don't see any problems with SetClippingRegion myself, but I'm testing on wxGTK (could be MSW specific).

**Comment by RobinD42** [comment]
From the exception message it sounds like you are calling with these parameters: 

> ClippingRegion: (False, 0, 0, 1012, 740)

Rather than like this:

>            clip = (1,80,949,515)
>            dc.SetClippingRegion(*clip)

FTR, these are the C++ overloads for `SetClippingRegion`, and wxPython has the corresponding wrappers: 

```
    void SetClippingRegion(wxCoord x, wxCoord y, wxCoord width, wxCoord height);
    void SetClippingRegion(const wxPoint& pt, const wxSize& sz);
    void SetClippingRegion(const wxRect& rect);
    void SetDeviceClippingRegion(const wxRegion& region);
```

**Comment by sancelot** [comment]
This is MSW specific

**Comment by swt2c** [comment]
Can you provide a small, self-contained example that reproduces the problem?

**Comment by mesalu** [comment]
@sancelot either the error you copy-pasted is NOT correct (which I find unlikely), or the snippet you've provided did not cause the error. 
Please follow the full stack trace to the real point of error in your code and provide a minimal functioning replication of it, to get help.

As presented this looks like improper use of the function (as indicated by the parameters described in the trace back, and the fact that the local `clip` doesn't match the example OR the parameters described later on), and not a bug with Phoenix. If the line of code throwing the exception really, truly, is just `dc.SetClippingRegion(*clip)`, I advise that you be triply sure that `clip` is what composed of what you think it is, as I doubt it is set directly above the function call as presented to us.

**Comment by sancelot** [comment]
No. This really fails as explained. reverting To wxpython 4.0.7post2 the problèm does not occur. however I can t tell at the moment which dc class buffer has the problem. it sounds  like an overloaded class problem. I Will try To take time To isolate thé problèm in a unit test
